modules/manually-restoring-cluster-etcd-backup.adoc

```diff
@@ -0,0 +1,298 @@
+// Module included in the following assemblies:
+//
+// * disaster_recovery/scenario-2-manually-restoring-cluster-etcd-backup.adoc
+// * post_installation_configuration/cluster-tasks.adoc
+
+
+:_content-type: PROCEDURE
+[id="manually-restoring-cluster-etcd-backup_{context}"]
+= Restoring a cluster manually from an etcd backup
+
+The restore procedure described on the document "Restoring to a previous cluster state": 
+
+* Requires the complete recreation of 2 control plane nodes, which might be a complex procedure for clusters installed with the UPI method, since an UPI installation does not create any MachineSet for the control plane nodes. 
+* Uses the script `/usr/local/bin/cluster-restore.sh`, which starts a new single-member etcd cluster, then scales it to 3 members. 
+
+The procedure described in this document, instead:
+
+* Doesn't require to recreate any control plane node.
+* Directly starts a 3-members etcd cluster.
+
+If your cluster uses instead a control plane machine set, it is still suggested to use the "Restoring to a previous cluster state" document for a simpler etcd recovery procedure.
```

Comment:
I propose to remove the "your", just add "If the cluster uses". Additionally the sentence doesn't make sense.
Better would be "If the cluster uses the MachineSet for control plane, it is suggested to use...".

Reply:
Ack.